
CUDA Environment Info

-- JOB METADATA --
slurm job id: 2421039
batch size: 3000
learning rate: 0.010
epochs: 30

loading packages..
TensorFlow version 2.6.0
Keras version 2.6.0

loading data..
X shape: (355887, 1053, 4)
Y shape: (355887, 1053, 1)
Size of batches: 0.05 GB

Number of training samples: 249121
Number of validation samples: 53383
Number of test samples: 53383
Y train: (249121, 1053, 1)
X train: (249121, 1053, 4)


Traceback (most recent call last):
  File "scripts/pnet_train_loop.py", line 114, in <module>
    pnet_omicron = PointNet_omicron(shape=(X.shape[1], 4),
  File "/localscratch/russbate.2421039.0/scripts/pnet_models.py", line 218, in PointNet_omicron
    vector_dense_1 = layers.Dense(
  File "/usr/local/lib/python3.8/dist-packages/keras/engine/base_layer_v1.py", line 745, in __call__
    self._maybe_build(inputs)
  File "/usr/local/lib/python3.8/dist-packages/keras/engine/base_layer_v1.py", line 2068, in _maybe_build
    self.build(input_shapes)
  File "/usr/local/lib/python3.8/dist-packages/keras/layers/core.py", line 1187, in build
    self.bias = self.add_weight(
  File "/usr/local/lib/python3.8/dist-packages/keras/engine/base_layer_v1.py", line 423, in add_weight
    variable = self._add_variable_with_custom_getter(
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/training/tracking/base.py", line 813, in _add_variable_with_custom_getter
    new_variable = getter(
  File "/usr/local/lib/python3.8/dist-packages/keras/engine/base_layer_utils.py", line 117, in make_variable
    return tf.compat.v1.Variable(
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/ops/variables.py", line 266, in __call__
    return cls._variable_v1_call(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/ops/variables.py", line 212, in _variable_v1_call
    return previous_getter(
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/ops/variables.py", line 205, in <lambda>
    previous_getter = lambda **kwargs: default_variable_creator(None, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/ops/variable_scope.py", line 2612, in default_variable_creator
    return resource_variable_ops.ResourceVariable(
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/ops/variables.py", line 270, in __call__
    return super(VariableMetaclass, cls).__call__(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/ops/resource_variable_ops.py", line 1602, in __init__
    self._init_from_args(
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/ops/resource_variable_ops.py", line 1740, in _init_from_args
    initial_value = initial_value()
  File "/usr/local/lib/python3.8/dist-packages/keras/initializers/initializers_v2.py", line 226, in __call__
    return tf.constant(
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/framework/constant_op.py", line 271, in constant
    return _constant_impl(value, dtype, shape, name, verify_shape=False,
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/framework/constant_op.py", line 288, in _constant_impl
    tensor_util.make_tensor_proto(
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/framework/tensor_util.py", line 508, in make_tensor_proto
    raise TypeError("Expected Tensor's shape: %s, got %s." %
TypeError: Expected Tensor's shape: (1024,), got (256,).
